src/python/doc/wasserstein_distance_user.rst

```diff
@@ -9,12 +9,21 @@ Definition
 
 .. include:: wasserstein_distance_sum.inc
 
-This implementation is based on ideas from "Large Scale Computation of Means and Cluster for Persistence Diagrams via Optimal Transport".
+Functions
+---------
+This implementation uses the Python Optimal Transport library and is based on
+ideas from "Large Scale Computation of Means and Cluster for Persistence
+Diagrams via Optimal Transport".
```

Comment:
I didn't see it on POT wasserstein review, but a link in the bibtex would be better to advertise the paper.

Reply:
Done.